circle.yml

```diff
@@ -5,12 +5,17 @@ machine:
     version:
       2.3.2
 
-  services:
-    - elasticsearch
-
   environment:
     CIRCLE_ENV: test
 
+dependencies:
```

Comment:
Not sure what you mean by "brittle configuration" and if you're actually asking me. I guess we reached the limits of my English skills! :p

But yeah, when I saw the tests were doing stuff not supported by ES in CircleCI, I decided to check the versions.